src/skills-builder/skills-builder-modal/view-results/ViewResults.jsx

```diff
@@ -17,38 +19,64 @@ const ViewResults = () => {
 
   const [selectedJobTitle, setSelectedJobTitle] = useState('');
   const [jobSkillsList, setJobSkillsList] = useState([]);
-  // eslint-disable-next-line no-unused-vars
-  const [courseRecommendations, setCourseRecommendations] = useState([]);
+  const [productRecommendations, setProductRecommendations] = useState([]);
+  const [selectedRecommendations, setSelectedRecommendations] = useState({});
   const [isLoading, setIsLoading] = useState(true);
+  const [fetchError, setFetchError] = useState(false);
 
   useEffect(() => {
-    const getJobs = async () => {
+    const getRecommendations = async () => {
       // fetch list of jobs with related skills
-      const jobInfo = await searchJobs(jobSearchIndex, careerInterests);
+      const jobInfo = await searchJobs('jobSearchIndex', careerInterests);
```

Comment:
Is this intentional? We aren't passing the index object in anymore? just the string `jobSearchIndex`?